examples/extension_list/agent_extension_policy.rego

```diff
@@ -0,0 +1,122 @@
+package agent_extension_policy
```

Comment:
Add copyright notice for each source file.

Reply:
will do.